java/src/jmri/SpeedStepMode.java

```diff
@@ -20,30 +20,38 @@
  */
 @javax.annotation.concurrent.Immutable
 public enum SpeedStepMode {
-    UNKNOWN("unknown", 1, 0.0f),
+    // NOTE: keep these up to date with xml/schema/locomotive-config.xsd
+    UNKNOWN("unknown", 1, 0.0f, "SpeedStepUnknown"),
     // NMRA DCC standard speed step modes.
-    NMRA_DCC_128("128", 126), // Remember there are only 126 non-stop values in 128 speed.
-    NMRA_DCC_28("28", 28),
-    NMRA_DCC_27("27", 27),
-    NMRA_DCC_14("14", 14),
+    NMRA_DCC_128("128", 126, "SpeedStep128"), // Remember there are only 126 non-stop values in 128 speed.
+    NMRA_DCC_28("28", 28, "SpeedStep28"),
+    NMRA_DCC_27("27", 27, "SpeedStep27"),
+    NMRA_DCC_14("14", 14, "SpeedStep14"),
     // Non-DCC speed step modes.
-    MOTOROLA_28("motorola_28", 28), // Motorola 28 speed step mode.
-    TMCC_32("tmcc_32", 32), // Lionel TMCC 32 speed step mode.
-    INCREMENTAL("incremental", 1, 1.0f);
+    MOTOROLA_28("motorola_28", 28, "SpeedStep28Motorola"), // Motorola 28 speed step mode.
+    TMCC_32("tmcc_32", 32, "SpeedStep32TMCC"), // Lionel TMCC 32 speed step mode.
+    INCREMENTAL("incremental", 1, 1.0f, "SpeedStepIncremental");
 
-    SpeedStepMode(String name, int numSteps) {
-        this(name, numSteps, 1.0f / numSteps);
+    SpeedStepMode(String name, int numSteps, String peopleKey) {
```

Comment:
Please change references to `peopleKey` or `peopleName` to `description`, since that is a common term for this kind of key elsewhere in our code.

Reply:
Done.

Reply:
@trainman419 
The code still has `peopleKey` instead of `description`. Please change this.

Reply:
Ooops. Fixed.

Reply:
Thanks.